### DISCUSSION TITLE
Final Group Assignment

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#29] [Assignments]
### Due 6/13 

Please use this thread to submit your [final group assignment](https://albertkun.github.io/23S-ASIAAM-191A/assignments/final/).

## Submission

- [Select a presentation slot on the final sheet](https://docs.google.com/spreadsheets/d/1ZAr9Hd9fmtevEGRadA_a4xl-AoXM8eZJ3VQFygXnT5o/edit#gid=924366777)
- Separately take the [individual group evaluation](https://forms.office.com/r/TaNcM3RckD)
- Post your group's repository and website link below:

ex.
Group Repo: https://github.com/akochaphum/my-awesome-group
Group website: https://akochaphum.github.io/my-awesome-group/index.html

**Comment by czhangy** [comment]
Group Repo: https://github.com/czhangy/aa191a-inequality
Group Website: https://czhangy.github.io/aa191a-inequality/index.html

**Comment by kmtsujimoto** [comment]
Group Repo: https://github.com/jamesfalkenroth/healthcare
Group Website: https://jamesfalkenroth.github.io/healthcare/index.html

**Reply by kmtsujimoto** [reply]
https://docs.google.com/presentation/d/1bI5cIQ0HjKgzQN9mR_MlLHWJRYzEW1Q1ANsfe90mz3k/edit#slide=id.p

**Comment by SageLuong** [comment]
Group Repo: https://github.com/DylanJWan/AA191ATransportation
Group Website: https://dylanjwan.github.io/AA191ATransportation/index.html
Presentation Slides: https://www.canva.com/design/DAFlk7NNCAU/ywB_0kwFR2A3QtYSdoMVtQ/edit?utm_content=DAFlk7NNCAU&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

**Comment by catherinehhu** [comment]
Group Repo: https://github.com/catherinehhu/asian-am-191a-project
Group Website: https://catherinehhu.github.io/asian-am-191a-project/index.html

**Comment by cassiuscarandang** [comment]
Group Repo: https://github.com/cassiuscarandang/aashousingandhomelessness
Group Website: https://cassiuscarandang.github.io/aashousingandhomelessness/
Slide Presentation: https://www.canva.com/design/DAFlBtrH7Pc/ZG1n22l_mjF1jCns7J-6XQ/edit?utm_content=DAFlBtrH7Pc&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

**Comment by qatle** [comment]
Group Repo: https://github.com/trinhqale/AM191A-group-project
Group Website: https://trinhqale.github.io/AM191A-group-project/

**Reply by stellakangj** [reply]
Slides: https://docs.google.com/presentation/d/1GXz9-JFCrQetKxqQ__KNPseQpebzPn_bYhPsIoZ0D3g/edit#slide=id.p5

**Comment by paige-lee** [comment]
Group Repo: https://github.com/chrysanthemma/foodinsecurity
Group website: https://chrysanthemma.github.io/foodinsecurity/index.html

**Comment by joanneluong** [comment]
Group Repo: https://github.com/joanneluong/AAS191A-SEA
Group Website: https://joanneluong.github.io/AAS191A-SEA/index.html

**Reply by joanneluong** [reply]
Slideshow: https://docs.google.com/presentation/d/18isQcYOiCp2TF6mzELZVrBFfY-ilDq0f/edit?usp=sharing&ouid=115017078297312608084&rtpof=true&sd=true

**Comment by angelicaalacantar** [comment]
Group Repo: https://github.com/danigold1020/asianam-191a-oceanpollution
Group Website: https://danigold1020.github.io/asianam-191a-oceanpollution/Project/index.html

**Reply by angelicaalacantar** [reply]
Slides: https://docs.google.com/presentation/d/1c2WJgE5zbpWVF6dbn1BELoL6VPMWZRuUtIF-1T0RnK0/edit?usp=sharing

**Comment by emai14** [comment]
Slides: https://docs.google.com/presentation/d/11D3s_l0LsU1oiPNZacCVWdxHu1TjGjGYKM3SnS_YdNk/edit#slide=id.gd933aedcbc_0_2009
Group Repo: https://github.com/rebeccaksuen/Educational-Attainment
Group Website: https://rebeccaksuen.github.io/Educational-Attainment/PEAS/index.html
